frontend/src/templates/ResendOtpButton/ResendOtpButton.stories.tsx

```diff
@@ -26,8 +29,9 @@ InProgress.args = {
   isDisabled: true,
 }
 
-export const Playground = () => (
-  <ResendOtpButtonContainer
-    onResendOtp={() => new Promise((res) => setTimeout(res, 800))}
-  />
+export const Playground: Story<ResendOtpButtonContainerProps> = (args) => (
```

Comment:
???

Reply:
lmao rubbish from trying to solve another PR LOL. but at least this makes it easier to use that story :P